samples/valueentity-counter/test/testkit.js

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Mocks the behaviour of a single Akka Serverless Event-sourced entity.
+ *
+ * Handles any commands and events, internally tracking the state and maintaining an event log.
+ *
+ * NOTE: Entity IDs are not handled, so all commands are assumed to refer to a single entity.
+ */
+export class MockEventSourcedEntity {
```

Comment:
Kind of surprising that the codegen is generating this objects. Shouldn't that be part of the SDK testkit and evolve together with it? 

I also think we should not call it a Mock. It's not mocking anything. This is quite similar to the EventSourced Testkit in Akka.

Reply:
I don't know if it has been merged into SDK testkit. Do we have a plan for that? GitHub issue? I know it was talked about.

Reply:
I'm not aware of any discussion about it.